utils/notify/publish_notification.py

```diff
@@ -45,3 +46,25 @@ def post_publish_update(self, version: str):
                 unfurl_links=False,
                 unfurl_media=False,
             )
+
+
+def opts():
+    parser = argparse.ArgumentParser(
+        description="Sends a notification about a new release of platform-helper to Slack"
+    )
+    parser.add_argument(
+        "--send-notifications", help="Enables/disables notifications", type=bool, default=True
+    )
+    parser.add_argument(
+        "--publish-version", help="Specifies the published version", type=str, default=None
+    )
+    return parser.parse_args()
+
+
+def send_publish_notification_version(options):
```

Comment:
I know it's a simple command, but think it's worth a unit test just to double check that the arguments above are being passed in properly. Looks fine from reading over, but...